distributions/hplc.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<project default="usage">
+
+    <target name="distribution">
+        <antcall target="gather_modules" />
+        <ant antfile="${basedir}/build.xml" target="package_redistributables">
+            <property name="dist_sub.dir" value="HPLC" />
+            <property name="skip.zipDistribution" value="false" />
+            <property name="extraFileIdentifier" value="-HPLC" />
+        </ant>
+    </target>
+
+    <target name="gather_modules">
+        <ant antfile="${basedir}/distributions/legacy.xml" target="gather_modules" />
+        <copy todir="${dist.modules.dir}">
+            <filelist dir="${staging.modules.dir}">
+                <file name="HPLC.module" />
```

Comment:
This distribution file (hplc.xml), this file name `HPLC.module`, and the `extraFileIdentifier` seem like they should not reference HPLC any longer.